presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogFilter.java

```diff
@@ -139,8 +185,9 @@ public Builder addGroupFilter(String groupName, Predicate<String> filter)
         }
 
         /**
-         * Add a filter to apply to a capturing group in
-         * the regular expression for log file lines
+         * Identical to {@link #addGroupFilter(String, Predicate)},
```

Comment:
And numbered groups won't do much good when you're matching against a user-specified regular expression. Removed.